[doc] Add training CNN model on MNIST (#13724)
Samsung/ONE: docs/runtime/training_cnn_on_mnist.md

class names[:10]  : ['T-shirt/top', 'Trouser', 'Pullover', 'Dress', 'Coat', 'Sandal', 'Shirt', 'Sneaker', 'Bag', 'Ankle boot']
class length      : 10
The data files are created!
$ tree out
out
├── fashion-mnist.test.input.100.bin
├── fashion-mnist.test.output.100.bin
├── fashion-mnist.train.input.1000.bin
└── fashion-mnist.train.output.1000.bin

0 directories, 4 files
```

## Prepare model

Prepare a simple neural network for trainng MNIST datasets using the `tf.keras` API. This code generates a `model.tflite` TFLite model file.

```python
import tensorflow as tf
import tensorflow_datasets as tfds

# Create a model
model = tf.keras.models.Sequential([
    tf.keras.layers.Flatten(input_shape=(28, 28)),
    tf.keras.layers.Dense(128, activation='relu'),
    tf.keras.layers.Dense(10)
])

# Donwload train dataset
ds = tfds.load('mnist', split='train', as_supervised=True)
ds = ds.batch(128)

model.compile(optimizer='adam', loss='mean_squared_error')
model.fit(ds, epochs=5)

converter = tf.lite.TFLiteConverter.from_keras_model(model)
tflite_model = converter.convert()

with open('model.tflite', 'wb') as f:
    f.write(tflite_model)
```

## Convert tflite model to circle model

> ONERT takes both the tflight model and the circle model. So you can use this tflight model file directly. To use the circle model instead of the tfleet model, you must convert it to the circle model using the 'onecc' tool, and 'onecc' can only be used in Ubuntu. When the ‘one-compiler’ debian package is installed on the ONE release page, it is installed in the ‘/usr/share/one/bin’ directory. Please refer to the help for detailed guidelines. The following command converts the tflight model into a circle model.

Reviewer: typo  "tflight model", "tfleet model"

Reviewer 2: @ys44kim , why don't you try to use `Add a suggestion` feature. :)

Reviewer: @lemmaa 
i don't know `add a suggestion`feature,  I will use it from next time.
thanks

Author: @ys44kim Oh, I often use sentences made in code.i for documentation. My original sentences used `tflite`, but the code.i modified this word because she doesn't know the word. 😄

> ONERT takes both the tflight model and the circle model. So you can use this tflight model file directly. To use the circle model instead of the tfleet model, you must convert it to the circle model using the 'onecc' tool, and 'onecc' can only be used in Ubuntu. When the ‘one-compiler’ debian package is installed on the ONE release page, it is installed in the ‘/usr/share/one/bin’ directory. Please refer to the help for detailed guidelines. The following command converts the tflight model into a circle model.

Reviewer 2: ```suggestion
ONERT takes both the tflight model and the circle model. So you can use this tflight model file directly. To use the circle model instead of the tflitemodel, you must convert it to the circle model using the 'onecc' tool, and 'onecc' can only be used in Ubuntu. When the ‘one-compiler’ debian package is installed on the ONE release page, it is installed in the ‘/usr/share/one/bin’ directory. Please refer to the help for detailed guidelines. The following command converts the tflight model into a circle model.
```

> ONERT takes both the tflight model and the circle model. So you can use this tflight model file directly. To use the circle model instead of the tfleet model, you must convert it to the circle model using the 'onecc' tool, and 'onecc' can only be used in Ubuntu. When the ‘one-compiler’ debian package is installed on the ONE release page, it is installed in the ‘/usr/share/one/bin’ directory. Please refer to the help for detailed guidelines. The following command converts the tflight model into a circle model.

Reviewer: I don't understand the meaning of "ONE release page" exactly. does it mean package, not page?
```suggestion
ONERT takes both the TFLite model and the circle model. So you can use this TFLite model file directly. To use the circle model instead of the TFLite model, you must convert it to the circle model using the 'onecc' tool, and 'onecc' can only be used in Ubuntu. When the ‘one-compiler’ debian package is installed on the ONE release package, it is installed in the ‘/usr/share/one/bin’ directory. Please refer to the help for detailed guidelines. The following command converts the tflight model into a circle model.
```

> ONERT takes both the tflight model and the circle model. So you can use this tflight model file directly. To use the circle model instead of the tfleet model, you must convert it to the circle model using the 'onecc' tool, and 'onecc' can only be used in Ubuntu. When the ‘one-compiler’ debian package is installed on the ONE release page, it is installed in the ‘/usr/share/one/bin’ directory. Please refer to the help for detailed guidelines. The following command converts the tflight model into a circle model.

Reviewer 3: I thought that the release page meant here - https://github.com/Samsung/ONE/releases/tag/1.28.0 . 
But she might intend it differently. Let's wait her reply.

> ONERT takes both the tflight model and the circle model. So you can use this tflight model file directly. To use the circle model instead of the tfleet model, you must convert it to the circle model using the 'onecc' tool, and 'onecc' can only be used in Ubuntu. When the ‘one-compiler’ debian package is installed on the ONE release page, it is installed in the ‘/usr/share/one/bin’ directory. Please refer to the help for detailed guidelines. The following command converts the tflight model into a circle model.

Reviewer 4: (Optional)
```suggestion
ONERT takes both the tflight model and the circle model. So you can use this tflight model file directly. To use the circle model instead of the tfleet model, you have to convert it to the circle model using the 'onecc' tool, and 'onecc' can only be used in Ubuntu. When the ‘one-compiler’ debian package is installed on the ONE release page, it is installed in the ‘/usr/share/one/bin’ directory. Please refer to the help for detailed guidelines. The following command converts the tflight model into a circle model.
```

> ONERT takes both the tflight model and the circle model. So you can use this tflight model file directly. To use the circle model instead of the tfleet model, you must convert it to the circle model using the 'onecc' tool, and 'onecc' can only be used in Ubuntu. When the ‘one-compiler’ debian package is installed on the ONE release page, it is installed in the ‘/usr/share/one/bin’ directory. Please refer to the help for detailed guidelines. The following command converts the tflight model into a circle model.

Author: @ys44kim @zetwhite 

The ONE release page in the above document is correct @zetwhite said about the ONE release page.
The ONE release page meant `https://github.com/Samsung/ONE/releases/` url.

The v1.28.0 release only has a runtime package. If you look at the v1.27.0 release, you can find the compiler debian package. :)
